File: sdk/approute-sdk/golang/enum/result_code.go
package enum

// ResultCode represents the result code returned in every API envelope.
type ResultCode string

const (
	ResultOK                ResultCode = "OK"
	ResultAccepted          ResultCode = "ACCEPTED"
	ResultIdempotencyReplay ResultCode = "IDEMPOTENCY_REPLAY"
	ResultValidationError   ResultCode = "VALIDATION_ERROR"
	ResultUnauthorized      ResultCode = "UNAUTHORIZED"
	ResultForbidden         ResultCode = "FORBIDDEN"
	ResultNotFound          ResultCode = "NOT_FOUND"
	ResultConflict          ResultCode = "CONFLICT"
	ResultLimitReached      ResultCode = "LIMIT_REACHED"
	ResultOutOfStock        ResultCode = "OUT_OF_STOCK"
	ResultInsufficientFunds ResultCode = "INSUFFICIENT_FUNDS"
	ResultUpstreamError     ResultCode = "UPSTREAM_ERROR"
	ResultInternalError     ResultCode = "INTERNAL_ERROR"
)

File: sdk/approute-sdk/php/src/Enums/ResultCode.php
<?php

declare(strict_types=1);

namespace AppRoute\Sdk\Enums;

enum ResultCode: string
{
    case OK = 'OK';
    case ACCEPTED = 'ACCEPTED';
    case IDEMPOTENCY_REPLAY = 'IDEMPOTENCY_REPLAY';
    case VALIDATION_ERROR = 'VALIDATION_ERROR';
    case UNAUTHORIZED = 'UNAUTHORIZED';
    case FORBIDDEN = 'FORBIDDEN';
    case NOT_FOUND = 'NOT_FOUND';
    case CONFLICT = 'CONFLICT';
    case LIMIT_REACHED = 'LIMIT_REACHED';
    case OUT_OF_STOCK = 'OUT_OF_STOCK';
    case INSUFFICIENT_FUNDS = 'INSUFFICIENT_FUNDS';
    case UPSTREAM_ERROR = 'UPSTREAM_ERROR';
    case INTERNAL_ERROR = 'INTERNAL_ERROR';
}
